backend/qgis_server.py

```diff
@@ -110,6 +111,16 @@ def upload_project(self, project_id, title, path):
         return result.json()['id']
 
     def delete_project(self, project_id):
+        """
+        Deletes a project from the server.
+
+        :param project_id: Project's ID in the server.
+        :type project_id: int or str
+        :raise Unauthorized: When the servers negates the credentials,
+        preventing to do the request
```

Comment:
Unauthorized: when the request is not successful because the server didn't accept the credentials.

Reply:
"... the server rejects the credentials, " may be more precise. What do you think @manelclos ?